views/index.handlebars

```diff
@@ -90,6 +90,10 @@
           </li>
         </ol>
 
+        <div class="question i18n" data-i18n="ftm-want-to-mail-question">Want to mail your masks?</div>
+        <p>Our partners at <a target='_blank' rel='noreferrer noopener'
```

Comment:
this 2nd phrase is missing i18n. You can insert as html including the `a` tag by prefix the key with `[html]` in the data attribute. e.g. `data-i18n="[html]ftm-my-key-name"`